cmd/bbolt/main.go

```diff
@@ -1702,15 +1712,15 @@ type page struct {
 	ptr      uintptr
 }
 
-// DO NOT EDIT. Copied from the "bolt" package.
 func (p *page) Type() string {
-	if (p.flags & branchPageFlag) != 0 {
+	// For now all flags are exclusive,so let's be strict with expectations.
+	if p.flags == branchPageFlag {
 		return "branch"
```

Comment:
I am thinking why not to define const? We can get all const defined in a common package, so as to be shared by both boltDB and CMDs. Of course, in a separate PR.

```
type PageType string
const (
    BranchPage PageType = "branch"
    ...
)
```

Reply:
Agree. The extraction of shared library starts here: https://github.com/etcd-io/bbolt/pull/361